extensions/guacamole-auth-radius/src/main/java/org/apache/guacamole/auth/radius/conf/RadiusAuthenticationProtocolProperty.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
```

Comment:
Double oops. Netbeans template.

Reply:
As with others, I think this is actually okay in the most recent code.